public/app/plugins/datasource/loki/components/monaco-query-field/getOverrideServices.ts

```diff
@@ -0,0 +1,112 @@
+import { monacoTypes } from '@grafana/ui';
```

Comment:
This file is exactly the same as https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/prometheus/components/monaco-query-field/getOverrideServices.ts. I considered removing it and importing it from there, or something similar.